config.toml

```diff
@@ -1,3 +1,4 @@
+baseURL = "https://ipfs.io/blog"
```

Comment:
Will this affect the rendering if the blog is rendered from a ipfs.io/ipfs/QmHash..?